NSS-Day-Cohort-42/rare-media-moguls
# Get Posts by Category // Approving a Post works

#### Changes Made
1. Files Modified:
>PostList.js(Line 53 passing through props)
>PostRoutes.js (Added path)
>AdminPostApproval.js(Logic to keep posts sorted by categories even after approving)
>Category.js (Deleted double code)


#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout "mc-fresh"  ...sounds like a dj name.
```
2. Run the application.
```
npm start
```
3. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py server
```
4. Test app functionality.
> Log in as  username: "marymogul" password: "me"
> Click the category link on a post and see if it sorts by category.
> Approve/Unapprove a post and see if the list STAYS sorted.
5. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
